### PR TITLE
[ernie-relative] fix & add OpMapper for paddle squeeze2

### PR DESCRIPTION
**Pull request by Aganlengzi**
add OpMapper for paddle squeeze2 as CINN squeeze has been added.
fix squeeze with negative axes and add related unittests.

bcaf5e5c309d862f3a6a75c4f00fc84e

### CODE REVIEW

**Review by thisjiang — DISMISSED** [review]
LGTM

File: cinn/frontend/op_mappers/paddle/CMakeLists.txt [hunk 1]
```diff
@@ -1,23 +1,4 @@
...
-    reshape.cc
-    tanh.cc
-    matmul.cc
-    compare.cc
-    log.cc
-    concat.cc
-    constant.cc
-    where.cc)
+
+file(GLOB paddle_op_mapper_srcs LIST_DIRECTORIES false RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} *.cc)
```
Comment (wzzju) [inline cinn/frontend/op_mappers/paddle/CMakeLists.txt]: 不要这样写，后面若加单测，这样就不对了。

Comment (Aganlengzi) [reply cinn/frontend/op_mappers/paddle/CMakeLists.txt]: 已改回

**Review by wzzju — APPROVED** [review]
LGTM

**Pull request merged**